webrtc/RTCDataChannel-binaryType.window.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const validBinaryTypes = ['blob', 'arraybuffer'];
+const invalidBinaryTypes = ['jellyfish', 'arraybuffer ', '', null, undefined];
+
+for (const binaryType of validBinaryTypes) {
+  test((t) => {
+    const pc = new RTCPeerConnection();
+    t.add_cleanup(() => pc.close());
+    const dc = pc.createDataChannel('test-binary-type');
+
+    assert_equals(dc.binaryType, "blob");
```

Comment:
We already check that when creating a data channel. I don't think it's needed here - it distracts from what we want to test.

Reply:
We should delete this line. Youenn is filing a spec bug on changing the default, given implementation state.

Reply:
Err... not sure I agree to be honest but we can discuss that in https://github.com/w3c/webrtc-pc/issues/2170.